### DISCUSSION TITLE
ebus Adapter 3 Wifi - "no signal"

### DISCUSSION THREAD

**Discussion by Juergen2020** [john30/ebusd#982] [Q&A] [answered]
Hello,
ich wollte nun endlich meinen ebus Adapter 3 an meiner Vaillant Wärmepumpe in Betrieb nehmen.
nach ein paar kleineren Hürden (div. Dokus und Links sind nicht mehr verfügbar, Links broken) habe ich den Adapter am laufen (mit Build 20221215).
fixe IP.
Aber "eBUS signal: no signal".
Habt ihr einen Tipp wo ich zum Suchen, Debuggen anfangen kann/soll? müssen Jumper besonders gesetzt sein? muss doch auch ohne Connection vom ebusd funktionieren oder?
die LEDs blinken nach Anschluss von ebus-Kabel. nehme an, dass hier also Traffic kommt.

danke
lG
Jürgen

**Comment by Koky05** [comment]
"eBUS signal: no signal" it indicated that you do not have ebusd running or connected to adapter. It is not about connection to Vaillant heatpump.

**Reply by Juergen2020** [reply]
"no signal" means no ebusd?
ok i have ebusd running.

how can i check if ebus traffic arrives?

 ebusctl i
version: ebusd 23.2.23.2
update check: OK
device: 192.168.0.45:9999
signal: acquired
symbol rate: 23
max symbol rate: 29
scan: finished
reconnects: 0
masters: 1
messages: 11
conditional: 0
poll: 0
update: 4
address 31: master #8, ebusd
address 36: slave #8, ebusd

**Comment by john30** [comment, marked as answer]
`signal: acquired` tells you that eBUS is alive
